internal/pally/global_test.go

```diff
@@ -0,0 +1,38 @@
+package pally_test
```

Comment:
@nomis52 This is what I expect UNS's use of this package to look like (someday).